### PR TITLE
Add RPC option to bypass agent

### PR DESCRIPTION
**Pull request by Aestek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,16 +33,27 @@ If you want to immediatly deregister them, use `consul-bench -service my-service
...
     	If -register is given, flap each instance between critical and passing state on given interval
+  -query-stale
+    	Run stale blocking queries
+  -query-wait duration
+    	Bloquing queries max wait time (default 10m0s)
   -register int
     	Register N -service instances
+  -rpc
+    	Use RPC server calls instead of agent HTTP
+  -rpc-addr string
```
Comment (pierresouchay) [inline README.md]: We can simplify this:
```
consul-bench [-dc datacenter_for_rpc_calls] [-deregister] [-query-stale] \
                         [-query-wait duration] [-register int] [-service string] [-token string]
                         [-watchers int]
                         <url_of_consul>
```
if URL starts with rpc://<server_address>[:port] -> then use RPC mode, else http

Comment (Aestek) [reply README.md]: Actually we still need the http addr when using rpc for registration and stuff like that

Comment (pierresouchay) [reply README.md]: huh... ok then

**Comment by Aestek** [comment]
I'll rework this to have sub commands as you suggested, this is getting a bit messy.

**Pull request merged**